Bottom: 701304925a9dfbf3a63c61c7a8a59d5e2360d6b7
Top:    84d06ae4a4863af87357b5141f286c3d10659379
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-21 23:23:06 +0000

Refresh of polygon-c-for-some-reason-the

---

diff --git a/src/polygon.c b/src/polygon.c
index 764fe8b..0ce7b4b 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -483,7 +483,7 @@ frac_circle (PLINE * c, Coord X, Coord Y, Vector v, int fraction)
   e1 = (v[0] - X) * POLY_CIRC_RADIUS_ADJ;
   e2 = (v[1] - Y) * POLY_CIRC_RADIUS_ADJ;
 
-  /* XXX: *//* NB: the caller adds the last vertex, hence the -1 */
+  /* XXX: */ /* NB: the caller adds the last vertex, hence the -1 */
   range = POLY_CIRC_SEGS / fraction;
   for (i = 0; i < range; i++)
     {
